bokeh/plotting.py

```diff
@@ -410,13 +410,13 @@ def output_file(filename, title="Bokeh Plot", autosave=False, mode="inline", roo
         print("Session output file '%s' already exists, will be overwritten." % filename)
 
 
-def show(obj=None, browser=None, new="tab", url=None):
-    """ 'shows' a plot object or the current plot, by auto-raising the window or tab
-    displaying the current plot (for file/server output modes) or displaying
-    it in an output cell (IPython notebook).
+def show(obj, browser=None, new="tab", url=None):
+    """ 'shows' a plot object, by auto-raising the window or tab displaying the
+    current plot (for file/server output modes) or displaying it in an output
+    cell (IPython/Jupyter notebook).
 
     Args:
-        obj (Widget/Plot object, optional): it accepts a plot object and just shows it.
+        obj (Widget/Plot object): requires a plot object and just shows it.
```

Comment:
>I should really write it up somewhere

Yep...

> nominative descriptions without a predicate.

Well, for me a plot object is a predicate, I mean, I (subject) display (verb) a plot object (predicate)... or there is something in English working different (sorry if I am asking a primary school level question).